chef-zero.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_dependency "hashie", ">= 2.0", "< 4.0"
   s.add_dependency "uuidtools", "~> 2.1"
   s.add_dependency "ffi-yajl", "~> 2.2"
-  s.add_dependency "rack", "< 2" # 2.0 requires Ruby 2.2+
```

Comment:
Rack being unstrained caused us a huge amount of pain and made it really hard / impossible to test older versions of chef/ohai where chef-zero came in without the constraint.  We need to have a constraint here to prevent 3.0 burning us again in the future.